frontend/components/CellInput.js

```diff
@@ -148,27 +152,29 @@ export const CellInput = ({
     variables_in_all_notebook,
 }) => {
     let pluto_actions = useContext(PlutoContext)
-
+    const client_id = pluto_actions.get_client_id()
```

Comment:
Can we make this a property instead of a function? Can still be on `pluto_actions` and added in an useMemo (e.g. `pluto_actions_with_client_id = React.useMemo(() => ({ ...pluto_actions, client_id: client_id }), [pluto_actions, client_id])` in Editor.js), feels off to have this part not participate in the React reactivity stuff